pdf-service.js

```diff
@@ -82,6 +82,16 @@ app.get('/from-url', async (req, res) => {
     }
 });
 
+function withHttp (url) {
+    if (!url) {
```

Comment:
I think we don't need this check, do we?

Reply:
If we don't have the url parameter it'll return an error in line 90 (
I can change url to "let", remove it by applying the function after url parameter validation, like this:

```javascript
    // change const to let
    let url = req.query.url;
    console.log('/from-url');

    if (!url) {
        let msg = 'missing parameter: \'url\'';
        deliverJson(res, {msg, params: req.params}, 400);
        return;
    }

    let pdfFilePath;
    try {
        // call the withHttp function before send to generatePdf()
        pdfFilePath = await generatePdf(withHttp(url), req.query.media);
```

Reply:
@fabriciojs 
What do you think? I like this new version of code, we check url parameter once